client/integration-tests/bean-validation/src/test/java/io/quarkiverse/openapi/generator/it/BeanValidationTest.java

```diff
@@ -0,0 +1,88 @@
+package io.quarkiverse.openapi.generator.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+
+import jakarta.inject.Inject;
+import jakarta.validation.Valid;
+import jakarta.validation.ValidatorFactory;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Pattern;
+
+import org.junit.jupiter.api.Test;
+import org.openapi.quarkus.bean_validation_false_yaml.api.UnvalidatedEndpointApi;
+import org.openapi.quarkus.bean_validation_false_yaml.model.UnvalidatedObject;
+import org.openapi.quarkus.bean_validation_true_yaml.api.ValidatedEndpointApi;
+import org.openapi.quarkus.bean_validation_true_yaml.model.ValidatedObject;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+class BeanValidationTest {
+
+    @Inject
+    ValidatorFactory validatorFactory;
+
+    @Test
+    void testValidationAnnotationsAreInPlaceApi() {
+        try {
+            Method method = ValidatedEndpointApi.class.getMethods()[0];
+            Annotation[][] annotations = method.getParameterAnnotations();
+            Boolean validationAnnotationExists = Arrays.stream(annotations)
+                    .allMatch(annotations1 -> Arrays.stream(annotations1)
+                            .filter(a -> a.annotationType().equals(Valid.class)).toList()
+                            .size() == 1);
+
+            assertThat(validationAnnotationExists).isTrue();
+        } catch (Exception e) {
+            fail();
+        }
+    }
+
+    @Test
+    void testValidationAnnotationsAreInPlaceModel() {
+        try {
+            Field id = ValidatedObject.class.getDeclaredField("id");
+            Field name = ValidatedObject.class.getDeclaredField("name");
+
+            assertThat(id.isAnnotationPresent(Max.class)).isTrue();
+            assertThat(name.isAnnotationPresent(Pattern.class)).isTrue();
```

Comment:
Besides the existing assertions, can you please add one assertion for each validation in `client/deployment/src/main/resources/templates/libraries/microprofile/beanValidationCore.qute`?

Reply:
@FelixScherzinger it seems like you missed this comment.

Reply:
Sorry, my bad. I just added assertions for all possible validation annotations.